package.json

```diff
@@ -75,7 +75,8 @@
       "at-rule-empty-line-before": null,
       "font-family-no-missing-generic-family-keyword": null,
       "string-quotes": "double",
-      "selector-pseudo-element-no-unknown": null
+      "selector-pseudo-element-no-unknown": null,
+      "no-descending-specificity": null
```

Comment:
Why is this rule added? Was it not a useful warning?

Reply:
no, it make code unreadable

Reply:
For example, you can declare :hover state on .input and .textarea in same time only if you don't modify .input or textarea later on... it's hard to explain..

Reply:
Easier with example: